src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Pages/CallWebApi.razor

```diff
@@ -0,0 +1,36 @@
+﻿@page "/callwebapi"
+@using Microsoft.AspNetCore.Authorization
+@using Microsoft.AspNetCore.Components.WebAssembly.Authentication
+@using ComponentsWebAssembly_CSharp.Shared
+@attribute [Authorize]
+@inject HttpClient Http
+
+<h1>Call Graph</h1>
```

Comment:
Shouldn't the title be different?

Reply:
Yes, good catch

Reply:
Fixed (in the other PR: https://github.com/dotnet/aspnetcore/pull/26547)